scss/bootstrap.scss

```diff
@@ -41,6 +41,7 @@
 @import "popover";
 @import "carousel";
 @import "spinners";
+@import "skeletons";
```

Comment:
I don't like it either, I used that name because it seems to be the known name for this css technique.
The `placeholder-*` word is an option, too long maybe?

`replace-*`
`mock-*`
`fake-*`
`skl-*`
`ph-*`
`marker-*`
`dash-*`
...

I let you decide 😋

Reply:
`placeholder` sounds better to me, it was known as "content placeholder" for a long time (eg in 2014, https://cloudcannon.com/deconstructions/2014/11/15/facebook-content-placeholder-deconstruction.html).